_acs-aem-commons/features/sling-model-injectors/shared-value-map-value/index.md

```diff
@@ -10,3 +10,30 @@ initial-release: 3.17.0
 ## Purpose
 
 Provides convenience for for accessing Shared Properties as implemented by ACS Commons [Shared Component Properties](/acs-aem-commons/features/shared-component-properties/index.html).
+
+## Example
+
+    @Model(adaptables = {SlingHttpServletRequest.class, Resource.class})
+    public final class SharedComponentPropsExampleModel {
+
+        @SharedValueMapValue(name = "titleText", type = SharedComponentProperties.ValueTypes.SHARED)
```

Comment:
Is the example purposely using `"titleText"` as the name for all?

Reply:
Well, yes - it aligns well with the example from [Shared Component Properties](https://adobe-consulting-services.github.io/acs-aem-commons/features/shared-component-properties/index.html). Compare with [shared-component-props-example.jsp](http://localhost:4502/crx/de/index.jsp#/apps/shared-component-properties-example/components/content/shared-component-props-example/shared-component-props-example.jsp) from [shared-component-props-example-1.0.zip](https://adobe-consulting-services.github.io/acs-aem-commons/features/shared-component-properties/shared-component-props-example-1.0.zip)

Reply:
It does, but in practice a sling model would likely only ever inject a particular field into a single variable (similar to standard dialog values being injected).  I cant think of a valid/common reason to inject a global, shared, and merged value for the same property - you would just use the single level that you truly care about (merged if you simply want the most specific value).

I would propose you import 3 different fields if you want to show all `ValueTypes`.

Also, its worth nothing that `name` is not needed if the variable matches, similar to normal `@ValueMapValue` injection.

Reply:
Well, it's just an example, so as such - it allows for some kind of flexibility different from production code, just to showcase the possibilities. Currently the only info in the docs simply links to [Shared Component Properties](https://adobe-consulting-services.github.io/acs-aem-commons/features/shared-component-properties/index.html) where the example introduces `titleText`, so for the sake of consistency it made sense. But yeah, less is more, simple is better, so I've pushed a simpler version of the example, anything not covered in it will be trivial to figure out on demand for any dev.